microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,47 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, either enable the feature locally in Microsoft Edge, or register in the Digital Goods API origin trial to test the API in production.
+
+
+<!-- ------------------------------ -->
+#### Enable the feature locally
+
+To enable the Digital Goods API on your developer machine only, for testing:
+
+TODO: How do you enable the flags `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`?  They are excluded from `edge://flags`.
+
+<!-- The Edge Experiments flag in `edge://flags/` will be enabled by default, when this feature ships in Edge Stable. -->
```

Comment:
which flag? 3 flags?  not surfaced in UI of /flags/?  
The Edge Experiments flag in `edge://flags/` will be enabled by default, when this feature ships in Edge Stable.
Delete these commented-out lines?  This PR will merge before end of OT.  Maybe delete these lines in this PR, and rewrite article when OT ends (future PR).
@runyuany

Reply:
Yes, the flags in question are `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`. 
I agree to delete these commented-out lines in this PR, thanks.

Reply:
deleted hidden comments and TODO line